File: Modul 1/Bahasa C/PRAK110-2310817210005-GALIH_AJI_SABDARAYA.c
#include <stdio.h>
#include <math.h>

int main(){
    int Alas = 5, Tinggi = 12;
    int Hipotenusa = sqrt(Alas * Alas + Tinggi * Tinggi);
    int Keliling_Segitiga = Alas + Tinggi + Hipotenusa;
    int Luas_Segitiga = Alas * Tinggi / 2;
    printf("Diketahui :\n");
    printf("Alas = %d cm\n", Alas);
    printf("Tinggi = %d cm\n\n", Tinggi);
    printf("Jawab :\n");
    printf("Sisi A = %d cm\n", Tinggi);
    printf("Sisi B = %d cm\n", Hipotenusa);
    printf("Sisi C = %d cm\n", Alas);
    printf("Keliling = %d cm\n", Keliling_Segitiga);
    printf("Luas = %d cm", Luas_Segitiga);
    return 0;
}

File: Modul 1/Bahasa Python/PRAK110-2310817210005-GALIH_AJI_SABDARAYA.py
import math
Alas = 5
Tinggi = 12
Hipotenusa = round(math.sqrt(Alas ** 2 + Tinggi ** 2))
Keliling_Segitiga = Alas + Tinggi + Hipotenusa
Luas_Segitiga = round(1/2 * Alas * Tinggi)
print("Diketahui :")
print(f"Alas = {Alas} cm")
print(f"Tinggi = {Tinggi} cm\n")
print("Jawab :")
print(f"Sisi A = {Tinggi} cm")
print(f"Sisi B = {Hipotenusa} cm")
print(f"Sisi C = {Alas} cm")
print(f"Keliling = {Keliling_Segitiga} cm")
print(f"Luas = {Luas_Segitiga} cm")